source/threads.tex

```diff
@@ -3339,6 +3341,13 @@
 \pnum\effects Unblocks all threads that are blocked waiting for \tcode{*this}.
 \end{itemdescr}
 
+\pnum
+\begin{note}
+If any of the \tcode{wait} functions exits via an exception, it is
+unspecified whether the \tcode{Lock} is held. One can use a \tcode{Lock} type
+that allows to query that, such as the \tcode{unique_lock} wrapper.
+\end{note}
```

Comment:
I'm not entirely comfortable positioning this between the `notify*` functions and the `wait*` functions -- it seems pretty easy to miss here.